### PR TITLE
Add LBK3 algo to yiimp

### PR DESCRIPTION
**Pull request by ETS5**

### CODE REVIEW

**Comment by ETS5** [comment]
What part of the Yaamp.php file failed or was that not tested?

**Comment by ETS5** [comment]
Thats my fault I tried to do this with about an hour of sleep and realized to late i removed some of the newer algos changes

**Comment by ETS5** [comment]
https://icemining.ca/
https://bsod.pw/site/mining
are pools that already have it up and running

File: stratum/stratum.cpp [hunk 1]
```diff
@@ -137,6 +137,7 @@ YAAMP_ALGO g_algos[] =
 	{"lyra2", lyra2re_hash, 0x80, 0, 0},
 	{"lyra2v2", lyra2v2_hash, 0x100, 0, 0},
 	{"lyra2z", lyra2z_hash, 0x100, 0, 0},
+	{"lbk3", lbk3_hash, 0x100, 0, 0},
```
Comment (tpruvot) [inline stratum/stratum.cpp]: an algo of this speed (in GHs) should not have a factor of 256

File: stratum/algos/makefile [hunk 2]
```diff
@@ -41,4 +41,4 @@ $(OUTPUT): $(OBJECTS)
 clean:
 	rm -f *.o
 	rm -f ar2/*.o
-	rm -f blake2/*.o
```
Comment (tpruvot) [inline stratum/algos/makefile]: please restore this line

Comment (ETS5) [reply stratum/algos/makefile]: Again I am sorry i was very tired when i did this. I have restored it

File: stratum/stratum.cpp (hunk 1 above)
Comment (ETS5) [reply stratum/stratum.cpp]: What factor should it be then?

Comment (tpruvot) [reply stratum/stratum.cpp]: yes, 1 is fine, but miners should have the same factor

File: web/yaamp/core/functions/yaamp.php [hunk 3]
```diff
@@ -26,6 +26,7 @@ function yaamp_get_algos()
 		'hex',
 		'hsr',
 		'lbry',
+		'lbk3'
```
Comment (tpruvot) [inline web/yaamp/core/functions/yaamp.php]: typo there

File: stratum/stratum.cpp (hunk 1 above)
Comment (tpruvot) [reply stratum/stratum.cpp]: ok... so

**Comment by tpruvot** [comment]
thanks for your time

**Comment by ETS5** [comment]
Thank you for your hard work and help!

**Pull request merged**